piknikki/slapjack-is-fun
# Feature/continue work on localstorage

*What does this PR do?*
- Fix some bugs and reset wholeDeck after someone wins.  

**Is this a feature or refactor?**
- feature, bug fix, will all need refactoring later.

**Problems Encountered & Resulting Solutions**
- I don't know why I can't reset wholeDeck to cardNames again after it's gone through one game. I also haven't worked out local storage yet. I'll do that tomorrow.

**How should this be tested?**
- Play the game. Should be able to do all the normal stuff and also start a new game. Only thing left is to actually update local storage correctly.

Any questions and possible steps moving forward.
- local storage needs to get done.
